101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
```

Comment:
@arroyc, I made the description more explicit with this commit: https://github.com/Azure/azure-quickstart-templates/pull/2984/commits/40dee94c549a362b89dd74affb622bac4b5c3dea